srcpkgs/xdg-desktop-portal-kde/template

```diff
@@ -11,13 +11,15 @@ makedepends="kf6-kio-devel kf6-kdeclarative-devel kf6-kirigami-devel
  kf6-kstatusnotifieritem-devel libplasma-devel kf6-kwayland-devel
  libepoxy-devel pipewire-devel glib-devel"
 depends="xdg-desktop-portal"
-checkdepends="dbus xdg-desktop-portal-kde python3-gobject"
+checkdepends="dbus python3-gobject"
 short_desc="Backend implementation for xdg-desktop-portal that is using Qt/KF6"
 maintainer="John <me@johnnynator.dev>"
 license="GPL-3.0-or-later"
 homepage="https://invent.kde.org/plasma/xdg-desktop-portal-kde"
 distfiles="${KDE_SITE}/plasma/${version}/${pkgname}-${version}.tar.xz"
-checksum=91e55a794f00fdb75305ed05f72a1ce310e228b115886f72471473c9c7c88960
+checksum=00bdf442d37b3080abfd2958425dd724a3a5019d50dfd7cb319e5160b27a6b05
+# check requires xdg-desktop-portal-kde, which leads to dependency cycle
+make_check=no
```

Comment:
Only `colorschemetest` requires this, just add `-E colorschemetest` after ctest

Reply:
Fixed!